src/components/Modals/Onboarding/OnboardingBasic.vue

```diff
@@ -199,16 +202,20 @@ export default defineComponent({
   },
   computed: {
     colleges(): Readonly<Record<string, string>> {
-      return Object.fromEntries(
-        Object.entries(reqsData.college).map(([key, { name }]) => [key, name])
-      );
+      const base = Object.entries(reqsData.college)
+        .filter(college => !college[0].startsWith('AS'))
+        .map(([key, { name }]) => [key, name]);
+      base.push(['AS', getCollegeFullName('AS')]);
+      base.sort((c1, c2) => c1[1].localeCompare(c2[1]));
+      return Object.fromEntries(base);
     },
     majors(): Readonly<Record<string, string>> {
       const majors: Record<string, string> = {};
       const majorJSON = reqsData.major;
+      const acr = this.collegeAcronym.replace('AS', 'AS2');
```

Comment:
is this arbitrarily choosing `AS2` to replace `AS` so that it passes the if statement?

Reply:
I think so! The major lists for `AS1` and `AS2` should be the same

Reply:
Confirmed this was the case in the `index.ts` in the data folder.